packages/react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -140,9 +140,16 @@ export class Suggestions<T> extends React.Component<ISuggestionsProps<T>, ISugge
       ? { styles: spinnerStyles }
       : { className: css('ms-Suggestions-spinner', legacyStyles.suggestionsSpinner) };
 
-    const noResults = () => {
-      return noResultsFoundText ? <div className={this._classNames.noSuggestions}>{noResultsFoundText}</div> : null;
-    };
+    const noResults = () => (
+      // This ID can be used by the parent to set aria-activedescendant to this
+      <div id="sug-noResultsFound">
```

Comment:
just to complicate things, this id actually needs to be on the same element that has `role="option"`, because browser a11y trees are picky 😅